test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -506,12 +507,14 @@ Function sys.int2fixeddecimaldiv(smallint,sys.smallmoney)
 Function sys.int2rowversion(smallint,integer,boolean)
 Function sys.int2varbinary(smallint,integer,boolean)
 Function sys.int4_fixeddecimal_cmp(integer,sys.fixeddecimal)
+Function sys.int4_numeric_cmp(integer,numeric)
 Function sys.int4fixeddecimaldiv(integer,sys.fixeddecimal)
 Function sys.int4fixeddecimaldiv(integer,sys.smallmoney)
 Function sys.int4int2_avg(bigint[])
 Function sys.int4int2_sum(bigint)
 Function sys.int4rowversion(integer,integer,boolean)
 Function sys.int8_fixeddecimal_cmp(bigint,sys.fixeddecimal)
+Function sys.int8_numeric_cmp(bigint,numeric)
```

Comment:
We need a int8 to numeric = condition view test case to avoid add this dependency

Reply:
added

Reply:
This is support function for operator class we are introducing. So this function will be invoked whenever btree search is executed. Not sure how can we add explicit tests for this.

Reply:
Yes you're right, this one can't be avoided